app/gui/src/project-view/components/ControlGroup.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
+import VNodes from '@/components/VNodes.vue'
+
+const slots = defineSlots<{ default(): any }>()
+</script>
+
+<template>
+  <div class="ControlGroup">
+    <div v-for="(item, i) in slots.default()" :key="i" class="control">
+      <VNodes :vnodes="item" />
+    </div>
```

Comment:
Why do we do this way, instead of just `<slot />`?

Reply:
That's because I needed to wrap each individual slot child node in its own `div`. The alternative would be to wrap each individual `ActionButton` into its own component, but that drastically complicates the usage of `ControlGroup` component.

Reply:
Ah, I see now.